src/components/Lineage/Graph/index.tsx

```diff
@@ -74,16 +79,25 @@ export function EgeriaLineageGraph(props: Props) {
   };
 
   useEffect(() => {
+
+    if (!Object.values(LINEAGE_TYPES).includes(lineageType)) {
```

Comment:
i believe this check should be done after the "fetchData" brought the data with the selectedNode, this way you will have the Node Type available and check if it has a lineageType

Reply:
if this check is done after the fetch then the fetch will fail due to the fact that the link is wrong

Reply:
how do you know the link is wrong?

Reply:
lineageType comes as a path parameter and if for example you put vertical-lineageee instead of vertical-lineage in the URL, the call will fail

Reply:
const uri = (lineageType: any) => `/api/lineage/entities/${guid}/${lineageType}?includeProcesses=true`;